src/components/ProjectSearch.js

```diff
@@ -170,7 +177,11 @@ export class ProjectSearch extends Component<Props, State> {
 
   inputOnChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
     const inputValue = e.target.value;
+    this.props.updateSearchStatus(statusType.initial);
```

Comment:
Each time the user types we're setting the status to initial?

Reply:
I added this line in later. Would really leave it to you.

By only changing it to initial, it signified to me that we're about to make a new search once we press enter.

Reply:
We should avoid the work either here or in the action if the status is already initial.